src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -543,10 +543,28 @@ public static String CalculateChecksum(InputStream in, ChecksumType checksumType
             }
         }
 
-        byte[] mdbytes = md.digest();
+        return checksumDigestToString(md.digest());
+    }
+    
+    public static String CalculateChecksum(byte[] dataBytes, ChecksumType checksumType) {
```

Comment:
Camel-case?

Reply:
I think the method should be `calculateChecksum()`

Reply:
got it. I just used the name of the existing methods. 
But I have renamed all three - done.